dashboard/src/shared/AxiosInstance.ts

```diff
@@ -25,15 +25,25 @@ export function createAxiosInterceptors(axios: AxiosInstance, store: Store<IStor
   axios.interceptors.response.use(
     response => response,
     e => {
+      const dispatch = store.dispatch as ThunkDispatch<IStoreState, null, Action>;
       const err: AxiosError = e;
+      if (
+        err.config.xsrfCookieName === "XSRF-TOKEN" &&
+        err.code === undefined &&
```

Comment:
this is interesting, so the HTTP status code is `undefined` in this case? Other errors would have an actual status code, right?

What happens if you don't have a network connection, do you also hit the same error?

Reply:
Thanks for checking those cases. What happens if you load the app, and then disconnect WiFi/network connection? I'd be worried that response would be undefined in this case too.

Reply:
I have tried that but I am using a local instance of minikube so I still have access to the services. In any case yes, it should be like the case in which I remove the ingress backend. If the response is undefined, we ask the user to reload and then the  "Connection refused" error (or similar) is shown.

Reply:
> If the response is undefined, we ask the user to reload and then the "Connection refused" error (or similar) is shown.

If this is the case, it'll be confusing if it says my session has expired and it's just because I am no longer connected to the internet. Can you test this on a remote cluster? We might want to tweak the message we show based on this.

Reply:
>Can you test this on a remote cluster?

Yes, I tested it and without connection you see the modal. I didn't think this was very confusing since we are just asking to reload the page and then the user gets a more detailed error but it's true that it may not be accurate. I will be more explicit in the message.